### ISSUE TITLE
Python Script - Get Interconnection Queue Data from All ISOs

### ISSUE THREAD

**Issue by APIKreative** [kmax12/gridstatus#275]
Hello,

I am attempting to run the following script using your Python module:

 
![image](https://github.com/kmax12/gridstatus/assets/47405829/3304c4f2-734a-4a21-8390-16f5eae71607)

But I am receiving the following error:

 
![image](https://github.com/kmax12/gridstatus/assets/47405829/e07a7813-b2b0-4686-9e09-315c18f544ba)

It appears that the headers that the script is being pointed to might have changed? Has your team run into this before and found a solution?

Many thanks!

**Comment by kmax12** [comment]
thanks this looks like an issue with PJM (based on stack trace). until this is fixed you should be able to access the other iso queue directly

for example 

```python
Python 3.10.8 (main, Nov 24 2022, 08:09:04) [Clang 14.0.6 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import gridstatus
>>> iso = gridstatus.ISONE()
>>> iso.get_interconnection_queue()
      Queue ID                                       Project Name Interconnecting Entity      County State                           Interconnection Location Transmission Owner Generation Type  Capacity (MW)  ...  I39  Dev  Zone  FS SIS  OS FAC  IA Project Status
0         1438                                                NaN                   None         NaN   NaN                           CMP 34.5 kV, Section 182                NaN             NaN            NaN  ...    N  NaN   NaN NaN   N NaN NaN NaN            NaN
1         1437                                    Battery Storage                   None  New London    CT  Montville to Gales Ferry 115kV, approximately ...                NaN             BAT         103.32  ...    N   PD    CT NaN   N NaN NaN NaN            NaN
2         1436                                    Battery Storage                   None  New London    CT                        115kV Uncasville Substation                NaN             BAT         300.00  ...    N   PD    CT NaN   N NaN NaN NaN            NaN
3         1435  Non-controllable objective ETU to enable addit...                   None  New London    CT                                                NaN                NaN             NaN        1200.00  ...    N   PD    CT NaN   N NaN NaN NaN            NaN
4         1434                                    Battery Storage                   None    Franklin    MA  Lat Long is 42.6254349775399, -72.568813789001...                NaN             BAT         160.00  ...    N   PD   WMA NaN   N NaN NaN NaN            NaN
...        ...                                                ...                    ...         ...   ...                                                ...                ...             ...            ...  ...  ...  ...   ...  ..  ..  ..  ..  ..            ...
1583         5                                         Brayton Pt                   None     Bristol    MA                       Brayton Point 345 kV Station              NGRID             NaN            NaN  ...    Y  NaN    RI NaN   Y NaN NaN NaN            NaN
1584         4                          EMI Dighton Power Project                   None     Bristol    MA                                            U6 Line              NGRID              NG            NaN  ...    Y  NaN  SEMA NaN   Y NaN NaN NaN            NaN
1585         3                         Androscoggin Energy Center                   None    Franklin    ME                              Jay 115 kV Substation                CMP          DFO NG            NaN  ...    Y  NaN    ME NaN   Y NaN NaN NaN            NaN
1586         2                                       EMI-Tiverton                   None     Newport    RI                    Near Tiverton 115 kV Substation              NGRID              NG            NaN  ...    Y  NaN  SEMA NaN   Y NaN NaN NaN            NaN
1587         1                                    Millennium+K117                   None   Worcester    MA                                         W 123 Line              NGRID          DFO NG            NaN  ...    Y  NaN   WMA NaN   Y NaN NaN NaN            NaN

[1588 rows x 31 columns]
```

**Comment by TrentonBush** [comment]
It looks like the PJM column name issue was resolved by #262 and this issue can be closed?

**Comment by APIKreative** [comment]
It works now. Many thanks!
